addons/loyalty/tests/test_loyalty.py

```diff
@@ -21,9 +20,26 @@ def setUpClass(cls):
             'reward_ids': [(0, 0, {})],
         })
 
+    def test_loyalty_program_default_values(self):
```

Comment:
Maybe also test a program creation through a test Form to make sure it has the same values ?